packages/appsync-emulator-serverless/__test__/httpSource.test.js

```diff
@@ -0,0 +1,44 @@
+const nock = require('nock');
+const httpSource = require('../httpSource');
+
+describe('httpSource', () => {
+  beforeEach(() => {
+    nock('http://localhost:3000')
+      .get('/api/users')
+      .reply(200, {
+        data: [{ name: 'Name1' }, { name: 'Name2' }],
+      });
+
+    nock('http://localhost:3000')
+      .post('/api/users', { id: '123ABC' })
+      .reply(200, { id: '123ABC' });
+  });
```

Comment:
Curious why you added this -- were you planning on testing a mutation via http?

Reply:
Perhaps it's unnecessary, but I wanted to test posts. The project I'm using this on doesn't have mutations on an HTTP source, so I didn't have a real life test.

```
it('HTTP POST', async () => {
```